lib/git_repository/cloning.rb

```diff
@@ -29,25 +29,25 @@ def clone_svn(url)
         fetch:    ':refs/remotes/git-svn'
     end
 
-    set_section %w( svn-remote svn ), options
+    set_section %w(svn-remote svn), options
     pull_svn
   end
 
   def pull
     with_head_change head_oid do
-      git_exec 'remote', 'update'
+      git_exec('remote', 'update')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.